_data/scala-releases.yml

```diff
@@ -0,0 +1,20 @@
+- category: current_version
+  title: Current 3.1.x release
+  version: 3.1.1
+  release_date: February 1, 2022
+- category: current_version
+  title: Current 2.13.x release
+  version: 2.13.8
+  release_date: January 12, 2022
+- category: maintenance_version
+  title: Latest 2.12.x maintenance release
+  version: 2.12.15
+  release_date: September 14, 2021
+- category: "maintenance_version"
+  title: "Last 2.11.x maintenance release"
+  version: 2.11.12
+  release_date: "November 9, 2017"
+- category: "maintenance_version"
+  title: "Last 2.10.x maintenance release"
+  version: 2.10.7
+  release_date: "November 9, 2017"
```

Comment:
We should probably update the Scala 2 and Scala 3 release instructions to point them to this file, which needs to be updated on every new release.

Reply:
@SethTisue @Kordyjan after this PR is merged this page will need updating on each Scala release, instead of `/download/scala2.md`